attributes/user.rb

```diff
@@ -4,4 +4,6 @@
   user['group'] = 'ajones'
   user['gid'] = '1000'
   user['home'] = '/home/ajones'
+  user['home']['manage?'] = true
```

Comment:
This needs to default to either `false` or `nil`, or it will break all existing users.